src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -336,7 +418,7 @@ export class StudyViewPageStore {
         _.reduce(this.clinicalAttributes.result, (acc: { [id: string]: ChartMeta }, attribute) => {
```

Comment:
the reduce method should be assigned to _chartMetaSet variable.  Then pass empty object as the initial reduce value.  TS will derive type from the acc param typing.